### PR TITLE
Document create folder

### PR DESCRIPTION
**Pull request by ginach**

### CODE REVIEW

File: docs/items.md [hunk 1]
```diff
@@ -67,6 +68,33 @@ await oneDriveClient
           .GetAsync();
 ```
 
+Create a folder
+-------------------------
+
+### 1. By POST
```
Comment (rgregg) [inline docs/items.md]: Can you call this one: 1. By POST to a known folder ID

File: docs/items.md [hunk 2]
```diff
@@ -67,6 +68,33 @@ await oneDriveClient
...
+var folderToCreate = new Item { Folder = new Folder(), Name = "folder name" };
+var createdFolder = await oneDriveClient
+          .Drive
+          .Items[itemId]
+          .Children
+          .Request()
+          .AddAsync(folderToCreate);
+```
+
+### 1. By PUT
```
Comment (rgregg) [inline docs/items.md]: And call this one: 2. By PUT with a given path

File: docs/items.md [hunk 3]
```diff
@@ -67,6 +68,33 @@ await oneDriveClient
...
+
+```csharp
+var folderToCreate = new Item { Folder = new Folder() };
+var createdFolder = await oneDriveClient
+          .Drive
+          .Items[itemId]
```
Comment (rgregg) [inline docs/items.md]: Change this to Drive.Root

File: docs/items.md [hunk 4]
```diff
@@ -67,6 +68,33 @@ await oneDriveClient
...
+          .ItemWithPath("folder name")
```
Comment (rgregg) [inline docs/items.md]: And make this be folder/subfolder

**Comment by peternied** [comment]
Looks good

**Pull request merged**